nbgrader/docs/source/configuration/jupyterhub_config.rst

```diff
@@ -29,94 +29,83 @@ the same system. By integrating with JupyterHub, nbgrader streamlines the
-tells JupyterHub where the root of the class files directory (where the formgrader
-should be run from) is in relation to where the notebook server will run.
-
-Then, to run the formgrader, we must set the auth token to be the same as the
-one that we ran JupyterHub with (and note, as before, that you should generate
-this auth token and keep it secret):
-
-.. code:: bash
-
-    export CONFIGPROXY_AUTH_TOKEN='foo'
-    export JPY_API_TOKEN=$(jupyterhub token --db=sqlite:///path/to/jupyterhub.sqlite -f /path/to/jupyterhub_config <user>)
-
-where ``<user>`` should be the username of a JupyterHub admin user,
-``sqlite:///path/to/jupyterhub.sqlite`` is the path to the JupyterHub database
-(e.g. ``sqlite:///jupyterhub.sqlite``), and ``/path/to/jupyterhub_config`` is
-the path to your ``jupyterhub_config.py`` file. This will store the API token
-in an environment variable called ``JPY_API_TOKEN``, which we'll use later.
-
-Now that we've set the auth token and the API token, we can launch the
-formgrader:
-
-.. code:: bash
-
-    nbgrader formgrade
-
-Running this command will launch the formgrader, which you should be able to
-access from ``localhost:8000/hub/nbgrader/example_course``. If you properly
-configured the ``notebook_url_prefix``, you should additionally be able to open
-live notebooks from the formgrader using your notebook server that was spawned
-by JupyterHub.
+special ``HubAuth`` authenticator class which knows how to talk to JupyterHub.
 
 Configuring SSL
 ~~~~~~~~~~~~~~~
 
-You should almost certainly be running JupyterHub with SSL support, as it is
-insecure to not do so. However, this requires a bit more additional
-configuration on the nbgrader side as well. In particular, you will need to change the ``HubAuth.hub_base_url`` config option to include ``https``:
-
-.. code:: python
+.. warning::
 
-    c.HubAuth.hub_base_url = "https://localhost:443"
+    As of nbgrader version 0.4.0, there is no longer any additional
+    configuration that needs to be done to run with JupyterHub SSL support.
 
 For a minimal working example of using JupyterHub with SSL, please see the
 ``nbgrader.tests.formgrader.manager.HubAuthSSLManager`` in the `nbgrader tests
 <https://github.com/jupyter/nbgrader/blob/master/nbgrader/tests/formgrader/manager.py>`__.
+
+Running JupyterHub and formgrader
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The way we have set up the formgrader is as a `managed JupyterHub service
+<http://jupyterhub.readthedocs.io/en/latest/services.html>`__. This means that
+JupyterHub will keep track of the formgrader process for us. So, to run both
+the formgrader and JupyterHub, we only need to actually run JupyterHub:
+
+.. code:: bash
+
+    jupyterhub
+
+Running this command should launch JupyterHub, which will then be accessible
+from ``localhost:8000``. It will also launch the formgrader, which you should
+be able to access from ``localhost:8000/services/formgrader``. If you
```

Comment:
If we change the name of the service in the config file we will need to change this as well.